### DISCUSSION TITLE
Importing and using a Detectron2 Model

### DISCUSSION THREAD

**Discussion by kirillmeisser** [deepjavalibrary/djl#3443] [Q&A]
Hi, I am working on an object detection project and am currently using Detectron2 to train some models. Detectron2 has an export function to TorchScript, [see here](https://detectron2.readthedocs.io/en/latest/tutorials/deployment.html). However, the exported model requires torchvision as a dependency for some of the operations. My question is, does DJL support this dependency? If not is there a way to add this dependency? Any help is greatly appreciated!

**Comment by frankfliu** [comment]
You might hit this issue: https://github.com/deepjavalibrary/djl/issues/818

You might want to convert your model to OnnxRuntime, and use DJL to load onnx model

**Reply by kirillmeisser** [reply]
Hi @frankfliu thank you for your quick response! I will try out the two approaches!
